alibi/explainers/tests/test_shap_wrappers.py

```diff
@@ -363,7 +363,6 @@ def get_column_ranks(X, ascending=False):
 ]
 
 
-# @pytest.mark.skip
```

Comment:
Please leave these in as they make it very easy to run specific tests when things break down without messing with the CLI.